libs/blocks/table/table.js

```diff
@@ -54,6 +61,16 @@ function handleHeading(headingCols) {
         const btnWrapper = btn.closest('P');
         buttonsWrapper.append(btnWrapper);
       });
+
+      const row1 = document.createElement('div');
+      const row2 = document.createElement('div');
+      const row1LastIdx = isPriceBottom ? 3 : 4;
+      [...elements].forEach((e, idx) => {
+        if (idx < row1LastIdx) row1.appendChild(e);
+        else row2.appendChild(e);
+      });
```

Comment:
nit - though I'm not sure that's actually clearer.  Up to you.
```suggestion
[...elements].forEach((e, idx) => {
  (idx < row1LastIdx ? row1 : row2).appendChild(e);
});
```